src/game/components/Animation.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef GAME_COMPONENTS_ANIMATION_HPP_
+#define GAME_COMPONENTS_ANIMATION_HPP_
+
+#include <filesystem>
+#include "ecs/Component.hpp"
+#include "raylib/model/Animation.hpp"
+
+namespace game
+{
+    namespace components
+    {
+        struct Animation : public ecs::Component {
+            raylib::model::Animation animation;
+
+            Animation(raylib::model::Animation &panimation) : animation(panimation) {}
+            Animation(std::filesystem::path &path) : animation(path) {}
```

Comment:
The structure can inherit from the class raylib::model::Animation

There will be no more variable and it can be used as a raylib::model::Animation.